modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -18,63 +17,69 @@ endif::[]
 |===
 |Name |Parameter |Description
 
-ifdef::virtualmachine[]
+ifdef::vmtemplate[]
```

Comment:
This now means that this text only appears in the Template wizard field, which doesn't seem right ("Template from which to create the virtual machine").
However it also no longer seems relevant to the VM wizard fields due to the workflow change. If true, we can remove this entirely.

Reply:
Template, Template provider, Template support - these 3 items display only if the user selects Create virtual Machine Template. Therefore, they are embedded within the conditional expressions.